data/index.html

```diff
@@ -136,38 +136,48 @@ <h5 class="text-center">USGS Topo Maps</h5>
+        <p class="readable">We have organized our datasets into different categories based on some ISO thingee to bring order to the <s>galaxy</s> SGID. These categories are represented by a unified set of icons across all the SGID's different forms.</p>
     </div>
     <div class="grid__col section">
+        <h2>SGID Index</h2>
+            <div class="grid__col--span-all grid__col--centered grid__col--s-hidden" style="width:568px">
+                <a href="{% link data/sgid-index/index.html %}">
+                    <img alt="SGID Index Table Search" class="text-center" src="{% link images/sgid-index.png %}">
+                </a>
+            </div>
+            <p class="readable">The SGID Index is a sortable and searchable table of GIS resources from AGRC and many other state agencies. It is a great resource to quickly find GIS data and resources important to you. The index lists the resource's ISO category, name, and primary data steward, along with a short description of the resource and a link to a service endpoint (if available). </p>
+            <p class="section-read-more">
+                <a href="{% link data/sgid-index/index.html %}" class="btn">Search the Index</a>
+            </p>
+    </div>
+    <div class="grid__col section alternate">
         <h2>The Open SGID</h2>
-        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing the datasets from the SGID which are also available from the <a href="http://opendata.gis.utah.gov/">AGRC Open Data</a>. It is a publicly-available, direct database connection that can be utilized on a multitude of GIS platforms to access Utah spatial data. A <a href="https://github.com/agrc/open-sgid">GitHub repository</a> is used as the primary source of information for the Open SGID, and to catalog and address issues with explanations and or troubleshooting techniques. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
+        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing our curated collection of GIS datasets. It is a publicly-available database connection that can be used in many GIS platforms to directly access Utah spatial data. You can find the most up-to-date information about the Open SGID, including troubleshooting tips and an issue tracker, in the <a href="https://github.com/agrc/open-sgid">GitHub repository</a>. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
```

Comment:
**thought**: this seems like a pigeon hole. 

> can be used in many GIS platforms

What about something more inclusive like

> can be used by any system able to connect to a database with great support from many GIS platforms